routes/validators/comment.js

```diff
@@ -27,8 +27,25 @@ const updateComment = {
 
 const getArticleCommentsQuery = { page: Joi.number().min(1) };
 
+const highlightedTextComment = {
+  comment: Joi.object()
+    .keys({
+      parentId: Joi.string().trim(),
+      body: Joi.string()
+        .required()
+        .trim()
+        .min(3)
+        .max(255),
+      highlightedText: Joi.string().trim().required(),
+      startPoint: Joi.number(),
```

Comment:
@NkFab Is it allowed to have a negative number as `startPoint`?

Reply:
@oesukam not  it is going to be positive numbers only